src/baseSchema.ts

```diff
@@ -0,0 +1,60 @@
+import type { Node, NodeSpec } from "prosemirror-model";
+
+export const baseEmbedSchema: NodeSpec = {
+  caption: {
+    group: "block",
+    content: "paragraph",
+    toDOM() {
+      return ["div", { class: "imageNative-caption" }, 0];
+    },
+    parseDOM: [{ tag: "div" }],
+  },
+  altText: {
+    group: "block",
```

Comment:
Great spot. This is intentional – I mention in the PR description but it might not be clear exactly where that applies, so useful to clarify here. The schema is at the moment static, but we aim to eventually generate it from a specification passed to the embed creator itself. To avoid an (even) large(r) PR, that'll be the next piece of work.

Re: groups, yes, we'll probably want to remove the 'group' property entirely, as these fields are in no way generic – they're explicitly tied to specific nodes in a particular position.

Reply:
Sorry I missed that, please disregard my other schema comments.